sim/mage/combustion.go

```diff
@@ -82,6 +85,9 @@ func (mage *Mage) registerCombustionSpell() {
 			},
 			OnTick: func(sim *core.Simulation, target *core.Unit, dot *core.Dot) {
 				dot.CalcAndDealPeriodicSnapshotDamage(sim, target, dot.OutcomeSnapshotCrit)
+				if mage.t13ProcAura != nil && !dot.IsActive() {
+					mage.t13ProcAura.SetStacks(sim, 0)
+				}
```

Comment:
It would be cleaner to put this in the `OnExpire` of the Aura config for the DoT.